code/modules/chemistry/Chemistry-Dispenser.dm

```diff
@@ -34,6 +36,8 @@
 
 	New()
 		..()
+		if (!src.dispensable_reagents)
+			src.dispensable_reagents = basic_elements
```

Comment:
\>not refactoring the default chem dispenser into a subtype of an abstract parent 😔

Reply:
but map changes aaaaa